sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/automl/image/image_search_space_utils.py

```diff
@@ -158,13 +165,43 @@ def _convert_sweep_dist_dict_to_str_dict(sweep_distribution: dict) -> dict:
     return sweep_distribution
 
 
+class ChoicePlusSchema(ChoiceSchema):
+    """ Choice schema that allows boolean values also"""
+    values = fields.List(
+        UnionField(
+            [
+                DumpableIntegerField(strict=True),
+                DumpableStringField(),
+                fields.Float(),
+                fields.Boolean(),
```

Comment:
Should fields.Boolean() be at the end ? Otherwise, some other value (like dict) might map to boolean ?